src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp

```diff
@@ -242,7 +269,9 @@ update_daemon_config(const std::string& host)
     throw xrt_core::system_error(std::errc::invalid_argument, "Missing '" + std::string(config_file) + "'.  Cannot update");
 
   cfg.host = host;
-  std::cout << boost::str(boost::format("%s\n") % cfg);
+  // update the configuration file
+  cfile << boost::str(boost::format("%s\n") % cfg);
```

Comment:
I cant believe we did not write into the actual file... Just opened it and looked at it haha good catch!